tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/legalize_to_linalg.cc

```diff
@@ -224,6 +224,286 @@ static bool HasCanonicalDimensionNumbers(
   return true;
 }
 
+//===----------------------------------------------------------------------===//
+// mhlo.Einsum conversion patterns.
+//===----------------------------------------------------------------------===//
+
+// Looks through a set of dimension that has been marked as Reduction axes
+// if it is found within the set, then we set it as "reduction", otherwise
+// we can label it as "parallel"
+SmallVector<StringRef, 3> GetEinsumLoopsAttrs(
+    const llvm::SmallSetVector<StringRef, 4>& input_ind,
+    const llvm::SmallSetVector<StringRef, 4>& reduction_dims) {
```

Comment:
nit: We don't need the namespace

Reply:
Since smallsetvector is not part of "mlir/Support/LLVM.h", we still need to use the llvm:: namespace. 
I can also add "using llvm:: SmallSetVector", on top of the file to remove the namespace. Any thoughts? :)